ods_ci/tests/Resources/CLI/MustGather/MustGather.resource

```diff
@@ -10,6 +10,12 @@ Resource         ../../Common.robot
 Get must-gather Logs
     [Documentation]    Runs the must-gather image and obtains the ODH/RHOAI logs
     ${output}=    Run process    tests/Resources/CLI/MustGather/get-must-gather-logs.sh     shell=yes
+    IF  ${output.rc} == 0
+        Log To Console     ${output.stdout}
+    ELSE
+        Log To Console     ${output.stderr}
+    END
```

Comment:
maybe you don't need the IF/ELSE? just try printing the output variable?

Reply:
changed, thanks @bdattoma

Reply:
removed the log to console because showing the output:

<result object with rc 0>

and its useless